spec/requests/admin/schools/participants_spec.rb

```diff
@@ -5,10 +5,9 @@
 RSpec.describe "Admin::Schools::Participants", type: :request do
   let(:admin_user) { create(:user, :admin) }
   let(:school) { create(:school) }
-  let(:school_cohort) { create(:school_cohort, school: school) }
-  let!(:ect_user) { create(:user, :early_career_teacher, school_cohort: school_cohort) }
-  let!(:mentor_user) { create(:user, :mentor, school_cohort: school_cohort) }
-  let!(:unrelated_user) { create(:user, :mentor) }
+  let!(:ect_profile) { create :participant_profile, :ect, school: school }
+  let!(:mentor_profile) { create :participant_profile, :mentor, school: school }
```

Comment:
This doesn't look right?

Reply:
Anything apart from `ect_profile` not being used, I don't see it! 00

Reply:
Shouldn't these take a `school_cohort`, not a school?

Reply:
Ah, after the refactor. Hmm, good question - both profiles do have `has_one :school, through: :school_cohort` so my guess is that by passing `school` instead of cohort FactoryBot will create a cohort (with a school) and then assigns the school through that cohort, effectively leaving us with some orphaned school records. It's not really an issue for the test, but would be nice to avoid this. :)